app/views/includes/analytics.nunjucks

```diff
@@ -11,11 +11,9 @@
     ga('send', 'pageview');
   </script>
 
+  <script src="{{ SITE_ROOT }}/js/custom-ga.js" type="text/javascript"></script>
```

Comment:
Move this script include to within the GA if statement as there isn't much point in having it load when GA isn't loaded